common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -301,12 +300,7 @@ public override int GetHashCode()
 
         public static bool operator ==(ClientWebSocketChannel left, ClientWebSocketChannel right)
         {
-            if (ReferenceEquals(left, null))
-            {
-                return ReferenceEquals(right, null);
-            }
-
-            return left.Equals(right);
+            return left is null ? right is null : left.Equals(right);
```

Comment:
Same here, why are we using the 'is' operator and not == ? Also I don't get the logic here. Are we just trying to check if these two are the same object (at least that is my understanding looking at the code) ? In that case, you can just return left.Equals(right).